pyspark_datasources/opensky.py

```
@@ -0,0 +1,350 @@
+
+import requests
+import time
+from datetime import datetime, timezone
+from typing import Dict, List, Tuple, Any, Optional, Iterator
+from dataclasses import dataclass
+from requests.adapters import HTTPAdapter
+from requests.packages.urllib3.util.retry import Retry
+from enum import Enum
+
+from pyspark.sql.datasource import SimpleDataSourceStreamReader, DataSource
+from pyspark.sql.types import *
```

Ruff (0.12.2)

`from pyspark.sql.types import *` used; unable to detect undefined names

(F403)